test/vidom/tree-test.js

```diff
@@ -0,0 +1,372 @@
+var fixtures = require('../fixtures')('vidom');
+
+var test = fixtures.test;
+var fail = fixtures.fail;
+
+describe('VIDOM compiler/Tree', function() {
+  it('should compile example code', function() {
+    test(function() {
+      block('b1').tag()(
+        elem('e1')('a'),
+        elem('e2').match(function() {
+          return this.ctx.hooray();
+        })(function() {
+          return apply({ a: 1 });
```

Comment:
It doesn't make sense. It should be something like `.def()` and `apply('default', ...)`. Is it copy-paste? /cc @miripiruni